scheduling/fcfs_scheduling.py

```diff
@@ -0,0 +1,74 @@
+"""
+First come first serve(FCFS) scheduling algorithm, as the name suggests,
+the process which arrives first, gets executed first, or we can say
+that the process which requests the CPU first, gets the CPU allocated first.
+"""
+
+
+def find_waiting_time(processes, n, bt, wt) -> None:
```

Comment:
Why does the caller need to provide `n` when it can easily be calculated inside this function?

What are the data types of these parameters?  Type hints should tell the caller this.